hs/app/reach/embed/sh/_common/_docker-compose/service-devnet-algo-postgres-db.yml

```diff
@@ -0,0 +1,6 @@
+algorand-postgres-db:
```

Comment:
I am nervous about this being a separate file rather than just having one list of containers for algo and another list of containers for eth/cfx